components/io_expander/esp_io_expander_tca9554/README.md

```diff
@@ -0,0 +1,39 @@
+# ESP IO Expander Chip TCA9554
+
+Implementation of the TCA9554 io expander chip with esp_io_expander component.
+
+| Chip             | Communication interface | Component name | Link to datasheet |
+| :--------------: | :---------------------: | :------------: | :---------------: |
+| TCA9554          | I2C                     | esp_io_expander_tca9554 | [datasheet](https://www.ti.com/lit/gpn/tca9554) |
+
+## Add to project
+
+Packages from this repository are uploaded to [Espressif's component service](https://components.espressif.com/).
+You can add them to your project via `idf.py add-dependancy`, e.g.
+```
+    idf.py add-dependency esp_io_expander_tca9554==1.0.0
+```
+
+Alternatively, you can create `idf_component.yml`. More is in [Espressif's documentation](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/tools/idf-component-manager.html).
+
+## Example use
+
+Creation of the component.
+
+```
+    esp_io_expander_handle_t io_expander = NULL;
+    esp_io_expander_new_i2c_tca9554(1, 0x20, &io_expander);
```

Comment:
Could you provide a #define for the I2C address?
I can see that TCA9554 has 3 pins for I2C address configuration. Could you add this information to the readme?

Reply:
Ok, I added #define for all possible addresses and added detailed information about that. PTAL [here](https://github.com/espressif/esp-bsp/pull/83/commits/51f2a11bbd9d3912e0992533545052dd50dd5ff9#diff-bcf624c69d44b03a4c6e2d930e19d928f1777f4ed624c5b41bc6ec78ae9684a8R34-R58).